src/include/daos_srv/daos_server.h

```diff
@@ -353,9 +353,11 @@ enum dss_ult_type {
 	/** aggregation ULT */
 	DSS_ULT_AGGREGATE,
 	/** drpc listener ULT */
-	DSS_ULT_DRPC,
+	DSS_ULT_DRPC_LISTENER,
 	/** miscellaneous ULT */
 	DSS_ULT_MISC,
+	/** drpc handler ULT */
+	DSS_ULT_DRPC_HANDLER,
```

Comment:
minor, better to move DSS_ULT_DRPC_HANDLER ahead to just after DSS_ULT_DRPC_LISTENER and before DSS_ULT_MISC